packages/lab/src/Flow/stories/SubFlow/Node.tsx

```diff
@@ -0,0 +1,77 @@
+import { useMemo } from "react";
+import { css, cx } from "@emotion/css";
+import { NodeProps as ReactFlowNodeProps } from "reactflow";
+import { theme } from "@hitachivantara/uikit-styles";
+
+import { useHvNode } from "../../hooks";
```

Comment:
Should be imported from `lab` package inside stories.

Reply:
I think this is missing from last commit

Reply:
Also, in stories, I think `theme` should come from the `core` package and not `styles` since users shouldn't need to use the `styles` package